src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -170,6 +184,14 @@ export interface SavedObjectsIncrementCounterField {
   incrementBy?: number;
 }
 
+/**
+ * @internal
+ */
+export interface SavedObjectsFindInternalOptions {
+  /** This is used for internal consumers that need to use a PIT finder but want to prevent extensions from functioning. */
+  disableExtensions?: boolean;
+}
```

Comment:
We use the SOR's PointInTimeFinder internally when searching for aliases and shared origins for saved objects, but we need to disable the extensions for that to function correctly.

Before, when we had SOC wrappers, the SOR's PointInTimeFinder did not have any of the wrapper functionality applied. This `disableExtensions` internal option preserves that behavior.